service_mesh/v2x/ossm-security.adoc

```diff
@@ -7,6 +7,20 @@ toc::[]
 
 If your service mesh application is constructed with a complex array of microservices, you can use {ProductName} to customize the security of the communication between those services. The infrastructure of {product-title} along with the traffic management features of {ProductShortName} can help you manage the complexity of your applications and provide service and identity security for microservices.
 
+.Before you begin
+
+If you have a project, add your project to the xref:../../service_mesh/v2x/installing-ossm.adoc#ossm-member-roll-modify_installing-ossm[`ServiceMeshMemberRoll` resource]. 
+
+[NOTE]
+====
+After you add the the namespace to the `ServiceMeshMemberRoll`, access to services or pods in that namespace will not be accessible to callers outside the mesh.
+====
+
+If you don't have a project, install the xref:../../service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc#ossm-tutorial-bookinfo-overview_deploying-applications-ossm[Bookinfo sample application] and add it to the `ServiceMeshMemberRoll` resource. The sample application helps illustrate security concepts.
+
+[id="ossm-security-smcp_{context}"]
+== Securing your control plane
```

Comment:
Is this heading in the right place?  It looks odd in the output having a heading with no content and then immediately after another heading.

Reply:
Good point. I think that was part of an earlier idea for a draft and I left it in.